README.md

```diff
@@ -1,6 +1,6 @@
 <img alt="PHP" src="https://img.shields.io/badge/php-%23777BB4.svg?&style=for-the-badge&logo=php&logoColor=white"/> <img alt="JavaScript" src="https://img.shields.io/badge/javascript%20-%23323330.svg?&style=for-the-badge&logo=javascript&logoColor=%23F7DF1E"/> <img alt="Python" src="https://img.shields.io/badge/python%20-%2314354C.svg?&style=for-the-badge&logo=python&logoColor=white"/> <img alt="CSS3" src="https://img.shields.io/badge/css3%20-%231572B6.svg?&style=for-the-badge&logo=css3&logoColor=white"/>
 
-<h1><img  align="right;" alt="Brand Logo " src="https://github.com/siv2r/kidney-exchange/blob/master/images/logo3.png?raw=true" width="25" height="25">&nbsp;Kidney Exchange Program</h1>
+<h1>&nbsp;Kidney Exchange Program</h1>
```

Comment:
what is `&nbsp`?

Reply:
i think its for space 
its already there should i remove it too ?

Reply:
Yes, you can remove it. Also, you can use `#` instead of the `h1` tag for the heading